client/src/components/user-management-component/UserManagementStyles.css

```diff
@@ -0,0 +1,8 @@
+.dashboard-container {
```

Comment:
Since there are only two classes in the CSS file, it would probably be best to just use MUI container instead of div just for the uniformity of the codebase avoiding the use of a dedicated CSS file where you could then use the sx styling
Or you can use styled-components similar to how Kevin did with his containers for the main dashboard
https://github.com/uoftblueprint/tces/blob/main/client/src/components/dashboard-component/index.styles.jsx

Reply:
Just changed it :)